python/setup.py

```diff
@@ -188,11 +188,12 @@ def find_version(*filepath):
 
 requires = [
     "aiohttp",
+    "aioredis",
     "click >= 7.0",
     "colorama",
     "filelock",
     "google",
-    "grpcio",
+    "grpcio >= 1.28.1",
```

Comment:
Why is that?

Reply:
The minimal stable version for AsyncIO API of grpcio.